microsoft-edge/webview2/reference/javascript/chrome.yml

```diff
@@ -0,0 +1,23 @@
+### YamlMime:TSType
+name: Chrome
+uid: 'WebView2Script!Chrome:class'
+package: WebView2Script!

```

Comment:
Clicking the above link on the rendered page takes me back to the top level page in the JS ref docs. I suggest we change the name to match that page.

Reply:
In the `YamlMime:TSType` schema a package name is required, and by default the title of the top level page is the `<name of the package> package` despite there isn't really a WebView2Script package. There does not appear to be a way to manually override the title with this particular schema. The office excel script team is experiencing the same scenario and take it as is.